installation-usage.md

```diff
@@ -1043,6 +1044,11 @@ data:
     enableUserWorkload: true
 ```
 
+Note that the above is not needed when deploying the Security Profiles Operator
```

Comment:
@sheriff-rh as mentioned in https://github.com/openshift/openshift-docs/pull/52698, once this PR is merged, we no longer have to document that the user workload monitoring is to be enabled.

Reply:
Thank you!